### ISSUE TITLE
JSON fixtures with unicode escape sequences are not parsed correctly

### ISSUE THREAD

**Issue by vschoettke** [cypress-io/cypress#964]
See the "Current behavior" for bug details.

I already boiled the problem down to an error in the jsonlint module that is used for parsing the JSON data in https://github.com/cypress-io/cypress/blob/a888f89d4d0dd4fad8e064e7cfdba3e6459d25fa/packages/server/lib/fixture.coffee#L90 and filed and issue there (zaach/jsonlint#102), but because that last commit is 3 years ago I don't know if it's still maintained.

Returning the result of `JSON.parse()` if no jsonlint error happened will fix the error while still keeping jsonlint as validator.

- Operating System: MacOS 10.13.1
- Cypress Version: 1.1.2
- Browser Version: not relevant

### Is this a Feature or Bug?

This is a bug.

### Current behavior:

A JSON file that contains unicode escape sequences (\uXXXX) in string values is not parsed correctly ("\uXXXX" results in "\\uXXXX"). 

### Desired behavior:

JSON fixture is parsed correctly.

### How to reproduce:

See https://github.com/vschoettke/cypress_fixture_parse_bug for an example

#### Test code:

fixtures/test_error.json
```js
{
    "name": "\u30d0\u30b0"
}
```

integration/test_spec.js
```js
describe("cypress should load json fixture with unicode specifier correctly", () => {
    it("should decode correctly", () => {
        cy.server();
        cy.fixture("test_error.json").then((data) => {
            const realData = require("../fixtures/test_error.json");
            expect(data.name).to.equal(realData.name);
        });
    });
});
```

**Comment by bahmutov** [comment]
Might switch to updated fork https://github.com/circlecell/jsonlint-mod or just use `JSON.parse`

**Comment by bahmutov** [comment]
Yeah, seems it is escaping the slash, so it does not process unicode JSON fixtures correctly.

**Comment by brian-mann** [comment]
Released in [`1.1.3`](https://docs.cypress.io/guides/references/changelog.html#1-1-3).
